designs/otis/src/front.mjs

```diff
@@ -0,0 +1,175 @@
+import { pluginCutlist } from '@freesewing/plugin-cutlist'
+import { pluginBundle } from '@freesewing/plugin-bundle'
+import { back } from './back.mjs'
+
+function draftFront({
+  options,
+  Point,
+  Path,
+  points,
+  paths,
+  Snippet,
+  snippets,
+  complete,
+  sa,
+  paperless,
+  measurements,
+  store,
+  macro,
+  part,
+}) {
+  const waist = store.get('waist')
+  const ease = store.get('ease')
+  const sizeFactor = store.get('sizeFactor')
+  const armhole = store.get('armhole')
+  const snapPlacketWidth = store.get('SnapPlacketWidth')
+
+  points.p0 = new Point(0, 0)
+  points.p0Cp1 = points.p0.shift(180, 69.64248775236683 * (ease + 1) * sizeFactor)
+  points.p1 = points.p0.shift(137.39365873630706, 132.32581976385896 * (ease + 1) * sizeFactor)
+  points.p1Cp1 = points.p1.shift(275.68130162148066, 95.6099938788659 * (ease + 1) * sizeFactor)
+  points.p1Cp2 = points.p1.shift(334.5320444890682, 74.98871362703433 * (ease + 1) * sizeFactor)
+  points.p2 = points.p0.shift(208.92817996505278, 145.1854377221146 * (ease + 1) * sizeFactor)
+  points.p2Cp2 = points.p2.shift(356.8730716127121, 43.74056201237602 * (ease + 1) * sizeFactor)
+  points.p3 = points.p0.shift(244.49016929788834, 295.05507494424813 * (ease + 1) * sizeFactor)
+  points.p3Cp1 = points.p3.shift(0, 18.248785709538428 * (ease + 1) * sizeFactor)
+  points.p4 = points.p0.shift(265.3666986326574, 309.97788995830655 * (ease + 1) * sizeFactor)
+  points.p4.x = points.p0.x - snapPlacketWidth
+  points.p5 = points.p4.shift(270, measurements.waist * options.snapPlacket * 2)
+  points.p6 = new Point(0, points.p5.y)
+
+  let diff = 10
+  let iter = 1
+  do {
+    paths.armhole = new Path().move(points.p1).curve(points.p1Cp1, points.p2Cp2, points.p2).hide()
+
+    const armholeLength = paths.armhole.length() * 0.85
+    diff = armholeLength - armhole
+
+    if (diff < -1 || diff > 1) {
+      points.p1 = points.p0.shift(
+        137.39365873630706,
+        132.32581976385896 * (ease + 1) * sizeFactor + diff
+      )
+    }
+    iter++
+  } while ((diff < -1 || diff > 1) & (iter < 100))
```

Comment:
This is using the `&` operator which is almost certainly not what you want, because it's a bitwise AND which expects two numbers, but the left part is always going to be a boolean.

You probably want the `&&` operator, which is the logical AND.

Please address this because it's a bug.

Reply:
Oops!